streaming/README.md

```diff
@@ -108,6 +108,27 @@ A new file is introduced: [stream.conf](stream.conf) (to edit it on your system
 `/etc/netdata/edit-config stream.conf`). This file holds streaming configuration for both the
 sending and the receiving Netdata.
 
+You can add a [global] section on the master server's [stream.conf](stream.conf) and activate
+additional slave statistics as follows.
+
+```
+[global]
+				activate stream statistics = yes
```

Comment:
Too much indentation here. Looks funky when rendered—Markdown respects all that whitespace.

```suggestion
    activate stream statistics = yes
```
```